libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/model/SpeechVolume.kt

```diff
@@ -2,11 +2,22 @@ package com.mapbox.navigation.ui.voice.model
 
 import androidx.annotation.FloatRange
 
+private const val MINIMUM_VOLUME_LEVEL = 0.0
+private const val MAXIMUM_VOLUME_LEVEL = 1.0
+
 /**
  * The state is returned if we change the speech volume.
- * @param level
+ * @param level volume level must be a value between [0.0..1.0]
  */
 data class SpeechVolume(
-    @FloatRange(from = 0.0, to = 1.0)
+    @FloatRange(from = MINIMUM_VOLUME_LEVEL, to = MAXIMUM_VOLUME_LEVEL)
     val level: Float
-)
+) {
+    init {
+        if (level !in MINIMUM_VOLUME_LEVEL..MAXIMUM_VOLUME_LEVEL) {
+            throw IllegalArgumentException(
+                "Level must be between [$MINIMUM_VOLUME_LEVEL..$MAXIMUM_VOLUME_LEVEL]"
+            )
+        }
+    }
```

Comment:
Added this for consistency with other features https://github.com/mapbox/mapbox-navigation-android/pull/4109#discussion_r593485535

cc @abhishek1508